lms/static/sass/features/_course-sock.scss

```diff
@@ -102,6 +102,12 @@
         color: theme-color("inverse");
         background-image: none;
         box-shadow: none;
+        cursor: pointer;
```

Comment:
this shouldn't be necessary unless its being disabled on links somewhere.  The reason why it wasn't showing as a pointer was because the HTML was invalid and the browser was confused.